restalchemy/storage/sql/filters.py

```diff
@@ -23,18 +23,19 @@
 from restalchemy.dm import filters
 from restalchemy.dm import types
 from restalchemy.storage.sql.dialect.query_builder import common
-from restalchemy.storage.sql import utils
+
 
 LOG = logging.getLogger(__name__)
 
 
 @six.add_metaclass(abc.ABCMeta)
 class AbstractClause(object):
 
-    def __init__(self, column, value_type, value):
+    def __init__(self, column, value_type, value, session):
         super(AbstractClause, self).__init__()
         self._value = self._convert_value(value_type, value)
         self._column = column
+        self._session = session
```

Comment:
Do we need to have session in clauses? Didn't find the usage yet

Reply:
fixed